packages/react-server-dom-vite/src/ReactFlightClientConfigBundlerViteServer.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+export function loadChunk(filename: string): Promise<mixed> {
+  return __vite_preload__(filename);
+}
```

Comment:
Since this file and the browser version are the same they don't really need to be duplicated. You can just implement loadChunk in the bundler config where it is called.